code-submissions-clustering-plugin/build.gradle.kts

```diff
@@ -11,28 +11,31 @@ dependencies {
     implementation(libs.dataframe)
 }
 
-open class BaseCLITask : RunIdeTask() {
+abstract class BaseCLITask : RunIdeTask() {
+    @get:Input
+    abstract val taskName: Property<String>
+
+    private val inputFile: String? by project
+    private val language: String? by project
+    private val outputDir: String? by project
+    private val binaryInput: String? by project
```

Comment:
It looks like you have missed a distance limit argument that is only used by `cluster` task